venv/connections.py

```diff
@@ -1,21 +1,29 @@
 from pyspark.sql import SparkSession
+from pyspark.sql import *
```

Comment:
it's a bad practice to import all modules at once, better import what you really need